dataland-e2etests/src/test/kotlin/org/dataland/e2etests/auth/JwtAuthenticationHelper.kt

```diff
@@ -34,8 +32,7 @@ class JwtAuthenticationHelper {
     }
 
     private fun getSingleValueFromJsonStringForKey(key: String, jsonAsString: String): String? {
-        val node: ObjectNode = objectMapper.readValue(jsonAsString, ObjectNode::class.java)
-        return node.get(key)?.toString()?.trim('"')
+        return JSONObject(jsonAsString).get(key)?.toString()?.trim('"')
```

Comment:
In another conversation @fsoffel has told me that:

"There is `JSONObject(message).getString(key)!` for extracting string properties or `JSONObject(message).get(key)!.toString()` for general values."

So why not use the first one here?  The method is only being used to retrieve "access_token" from a JSON, which is a string.
And the method name also implies that this method is made for strings inside of JSONs.

Reply:
Done